molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/requirements.txt

```diff
@@ -2,3 +2,4 @@ molo.core=={{cookiecutter.molo_version}}
 {% for requirement in cookiecutter.require %}
 {{requirement}}
 {% endfor %}
+django-extensions
```

Comment:
@miltontony It's not needed, it adds some custom extensions like `./manage.py show_urls`. I thought it's useful for dev, I can remove it If you see it's not necessary :)